brewtils/test/fixtures.py

```diff
@@ -215,7 +216,12 @@ def instance_dict(ts_epoch):
                 "user": "guest",
                 "password": "guest",
                 "virtual_host": "/",
-                "ssl": {"enabled": False},
+                "ssl": {
```

Comment:
This is to reflect what the response will look like from beergarden as of [beer-garden/1289](https://github.com/beer-garden/beer-garden/pull/1289).